packages/hierarchy-builder/src/test/internal/operators/grouping/BaseClassGrouping.test.ts

```diff
@@ -0,0 +1,298 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { expect } from "chai";
+import sinon from "sinon";
+import { HierarchyNode } from "../../../../hierarchy-builder/HierarchyNode";
+import * as baseClassGrouping from "../../../../hierarchy-builder/internal/operators/grouping/BaseClassGrouping";
+import { ECClass, IMetadataProvider } from "../../../../hierarchy-builder/Metadata";
+import { createGetClassStub, createTestNode, TStubClassFunc } from "../../../Utils";
+
+describe("BaseClassGrouping", () => {
+  const metadataProvider = {} as unknown as IMetadataProvider;
+  const ecClass = {
+    fullName: "TestSchema.TestParentClass",
+    label: "TestParentClass",
+  } as unknown as ECClass;
```

Comment:
I think it's better to define this in the tests that use it - otherwise, when reading a test, we have to scroll up to see what class is this

Reply:
Changed.